consensus/tracer.go

```diff
@@ -0,0 +1,56 @@
+package consensus
+
+import (
+	"github.com/tendermint/tendermint/config"
+	tmtDb "github.com/tendermint/tendermint/libs/db"
+	tmtState "github.com/tendermint/tendermint/state"
+
+	"github.com/lightstreams-network/lightchain/setup"
+	stdTracer "github.com/lightstreams-network/lightchain/tracer"
+)
+
+type tracer struct {
+	*stdTracer.Tracer
+}
+
+func newTracer(trc *stdTracer.Tracer) tracer {
+	return tracer{trc}
+}
+
+func (t *tracer) AssertPersistedInitStateDb(tmtCfg *config.Config, ntw setup.Network) {
+	if ! t.ShouldTrace {
```

Comment:
I see this much simpler than having two distinct structs only for the Null version of the tracer